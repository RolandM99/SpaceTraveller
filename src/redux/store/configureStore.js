import { createStore, combineReducers, applyMiddleware } from 'redux';
import logger from 'redux-logger';
import thunk from 'redux-thunk';
import rocketReducer from '../reducers';

const reducer = combineReducers({
  rocketReducer,
  // additional reducers could be added here
});

const store = createStore(
  reducer,
  applyMiddleware(logger, thunk),
);

export default store;
