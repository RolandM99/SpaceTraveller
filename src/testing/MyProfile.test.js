import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store/configureStore';
import Myprofile from '../components/MyProfile';

describe('MyProfile Component test', () => {
  it('Renders MyProfile', () => {
    const app = renderer.create(
      <Provider store={store}>
        <Myprofile />
      </Provider>,
    ).toJSON();
    expect(app).toMatchSnapshot();
  });
});
