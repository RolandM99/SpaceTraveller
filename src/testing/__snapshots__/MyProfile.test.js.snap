// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyProfile Component test Renders MyProfile 1`] = `
<div
  className="wrapper_profile"
>
  <div
    className="profile"
  >
    <h3>
      My Rockets
    </h3>
    <ul
      className="list"
    />
  </div>
  <div
    className="profile"
  >
    <h3>
      My Missions
    </h3>
    <ul
      className="list"
    />
  </div>
  <div
    className="profile"
  >
    <h3>
      My Daragons
    </h3>
    <ul
      className="list"
    />
  </div>
</div>
`;
